sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -230,6 +230,13 @@ case class AlterTableSetPropertiesCommand(
     isView: Boolean)
   extends RunnableCommand {
 
+  if (isView) {
+    properties.get(TimestampTableTimeZone.TIMEZONE_PROPERTY).foreach { _ =>
```

Comment:
is there even a meaning to set properties for any views? we should either drop this check, or have a more general check.

Reply:
[HiveQL](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Create/Drop/AlterView) explicitly allows properties in view; I've never used them, though.